docs/source/markdown/podman-system.1.md

```diff
@@ -15,9 +15,10 @@ The system command allows you to manage the podman systems
 | -------  | --------------------------------------------------- | ---------------------------------------------------------------------------- |
 | df       | [podman-system-df(1)](podman-system-df.1.md)        | Show podman disk usage.                                                      |
 | info     | [podman-system-info(1)](podman-info.1.md)           | Displays Podman related system information.                                  |
+| migrate  | [podman-system-migrate(1)](podman-system-migrate.1.md)| Migrate existing containers to a new podman version.                       |
 | prune    | [podman-system-prune(1)](podman-system-prune.1.md)  | Remove all unused container, image and volume data                              |
 | renumber | [podman-system-renumber(1)](podman-system-renumber.1.md)| Migrate lock numbers to handle a change in maximum number of locks.      |
-| migrate  | [podman-system-migrate(1)](podman-system-migrate.1.md)| Migrate existing containers to a new podman version.                       |
+| reset    | [podman-system-reset(1)](podman-system-reset.1.md)| Reset storage back to original state.  Remove all pods, containers, images, volumes |
```

Comment:
missing period at end.  Also line 19 while you're at it.

Reply:
Fixed